src/pico/i_video.c

```diff
@@ -91,7 +91,8 @@ static uint16_t ega_colors[] = {
 
 // todo temproarly turned this off because it causes a seeming bug in scanvideo (perhaps only with the new callback stuff) where the last repeated scanline of a pixel line is freed while shown
 //  note it may just be that this happens anyway, but usually we are writing slower than the beam?
-#define USE_INTERP PICO_ON_DEVICE
+//  previously PICO_ON_DEVICE but disabled to save scratch_x space
```

Comment:
I couldn't find the version of arm-none-eabi-gcc Graham was using to get everything to fit into scratch_x, so I just deleted some things. I want to be as performant as reasonably possible, but I'm not about to do any ASM debugging for this.